src/Compilers/Core/Portable/Symbols/ISymbol.cs

```diff
@@ -55,6 +58,7 @@ public interface ISymbol : IEquatable<ISymbol>
         /// </summary>
         string MetadataName { get; }
 
+#nullable disable
```

Comment:
📝 Skipping the `Containing*` properties made this pull request viable. I will not be addressing these properties in this pull request.